company_website/static/common/scripts/gtag_setup.js

```diff
@@ -0,0 +1,6 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){
+  dataLayer.push(arguments);
+}
+gtag('js', new Date());
+gtag('config', 'AW-607460288');
```

Comment:
Is this script anywhere called?

Reply:
Right below. You were wondering why it accepts arguments if none are declared in the definition? Well how about this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

Turns out it isn't obligatory to declare function arguments in the JS. 😛

Reply:
![Uploading what-kind-of-black-magic-sorcery-is-this.jpg…]()

Reply:
![what-kind-of-black-magic-sorcery-is-this](https://user-images.githubusercontent.com/26469852/90101204-8b1f0c00-dd3e-11ea-8489-bf44404f9a49.jpg)

Reply:
IKR? 😁